polygonal_search.ipynb

```diff
@@ -0,0 +1,213 @@
+{
```

Comment:
<p>To be more honest it also implements a simpler and faster algorithm, because it considers the convex case only. However, your reason about vectorization is much more important here!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/notebooks_lf/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Great point, I forgot to mention it!